openshift/templates/tileserv/tileserv.yaml

```diff
@@ -202,6 +172,8 @@ objects:
                     secretKeyRef:
                       name: ${APP_NAME}-${SUFFIX}-pguser-${APP_NAME}-${SUFFIX}
                       key: dbname
+                - name: TS_DEBUG
```

Comment:
Is there any performance impact when having ts_debug=true? Can we conditionally set this to true just for PR builds?

Reply:
Good catch, I think this can be removed. Though what's nice is that the actual SQL query tileserv runs against postgres is logged to console/kibana.